cpp/ps0008/problem. 8.cpp

```diff
@@ -38,11 +44,16 @@ int main() {
         } else {
             std::cout << "Wrong input, try again" << std::endl;
         }
-        }
+    }
```

Comment:
- [x] prompted user to input a key
- [x] repeated the prompting process unless the key is valid integer
- [ ] removing useless multiples of key value